pages/vi/faq.md

```diff
@@ -74,6 +75,11 @@
     - avoid checking for host keys with `ssh -o StrictHostKeyChecking=no <user>@<host>`
     - remove the exact key line of the offending host by looking in the output for `Offending key in /home/peter/.ssh/known_hosts:<linenumber>` and then with `sed -i <linenumber>d ~/.ssh/known_hosts` remove this exact line
 
+#### Q12: Why can't I ssh into my Raspberry Pi after installing a new treehouses image?
+
++ If you run into this error you will see a message like " WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!" when you try to ssh into your Raspberry Pi after installing a new treehouses image onto it. Thankfully this is an easy fix
++ To fix this tpye the command `cd` on your local machine. Then `cd .ssh`. Open the `known_hosts` file by typing `vim known_hosts`. Inside of this file you need to find your Raspberry Pi's ip address and delete that line since that is old known host for the previous treehouses image. Save the file. Now you should be able to ssh into your Raspberry Pi wihtout an issue.
```

Comment:
`cd` on your local machine. Then `cd ~/.ssh`

adding `~/` will make sure they're in the correct directory when they're on their local machine.

Reply:
@CalebProvost `cd` will automatically take you to `~/` so it's just repeating yourself and not necessary

Reply:
yup... I definitely forgot about that... my bad